app/scripts/controllers/build.js

```diff
@@ -67,72 +126,17 @@ angular.module('openshiftConsole')
         // context into the log-viewer directive.
         $scope.projectContext = context;
         $scope.logOptions = {};
-        DataService.get("builds", $routeParams.build, context).then(
-          // success
-          function(build) {
-
-            $scope.loaded = true;
-            $scope.build = build;
-            setLogVars(build);
-            var buildNumber = $filter("annotation")(build, "buildNumber");
-            if (buildNumber) {
-              $scope.breadcrumbs[2].title = "#" + buildNumber;
-            }
 
```

Comment:
The request flow may be easier to follow here, but I have no problem with the inline anonymous functions.  Gain is mostly in DRY.